modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/blueprint/search/request/body/contributor/HighlightSXPSearchRequestBodyContributor.java

```diff
@@ -40,7 +40,9 @@ public void contribute(
 		HighlightConfiguration highlightConfiguration =
 			configuration.getHighlightConfiguration();
 
-		if (highlightConfiguration == null) {
+		if ((highlightConfiguration == null) ||
```

Comment:
Do other contributors suffer from the same issue? Like Aggregations?

Reply:
I didn't find others to have the same issue